src/app/[id]/page.tsx

```diff
@@ -0,0 +1,239 @@
+                  >
+                    {item => (
+                      <AutocompleteItem
+                        key={item.catalogueId}
+                        textValue={item.name}
+                        endContent={
+                          <div className="text-xs text-gray-500">
+                            {item.catalogueId}
+                          </div>
+                        }
+                      >
+                        <div>
+                          {item.name}
+                          <div>
+                            {item.startDate && (
+                              <span className="text-xs text-gray-500">Fra {item.startDate}</span>
+                            )}
+                            {item.endDate && (
+                              <span className="text-xs text-gray-500"> til {item.endDate}</span>
+                            )}
+                          </div>
+                        </div>
+                      </AutocompleteItem>
+                    )}
+                  </Autocomplete>
+                </div>
+
+                <form onSubmit={() => void handleSubmit(onSubmit)()} className="flex flex-col gap-4">
+                  <div className="flex flex-row gap-4">
+                    <Controller
+                      name="title"
+                      control={control}
+                      rules={{required: 'Tittel er påkrevd'}}
+                      render={({ field }) =>
+                        <Input
+                          {...field}
+                          className="w-2/3"
+                          type="text"
+                          label="Tittel"
+                          isDisabled
+                        />
+                      }
+                    />
+                    <Controller
+                      name="titleId"
+                      control={control}
+                      rules={{required: 'ID er påkrevd'}}
+                      render={({ field }) =>
+                        <Input
+                          {...field}
+                          className="w-1/3"
+                          type="text"
+                          label="ID"
+                          isDisabled
+                        />
+                      }
+                    />
+                  </div>
+                  <Controller
+                    name="date"
+                    control={control}
+                    rules={{
+                      required: 'Dato er påkrevd',
+                      validate: {
+                        futureDate: value => {
+                          return value >= today('Europe/Oslo')
+                            ? 'Datoen kan ikke være i fremtiden'
+                            : true;
+                        }
+                      }
+                    }}
+                    defaultValue={dateValue(new Date(extractedMetadata.date))}
+                    render={({field}) => (
+                      <DateInput
```

Comment:
Using DatePicker allows both keyboard input and click on calendar, maybe it would be a better choice

Reply:
Updated with DatePicker!